lib/6to5/transformation/templates/get.js

```diff
@@ -0,0 +1,23 @@
+(function get(object, property, receiver) {
+  var desc = Object.getOwnPropertyDescriptor(object, property);
+
+  if (desc === void 0) {
+    var parent = Object.getPrototypeOf(object);
+
+    if (parent === null) {
+      return void 0;
+    } else {
+      return get(parent);
+    }
+  } else if ("value" in desc && "writable" in desc) {
```

Comment:
value needs to be checked with `in`, but `writeable` can merely be a property lookup `desc.writable !== undefined`